scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,273 @@
+# RUN export SBT_OPTS="-Dsbt.global.base=/path/to/another/tmp"
+
+########################### TEST-8 #############################
+
+# sbt build
+RUN whoami
+RUN env
+RUN pwd;ls -all
+RUN sbt sbtVersion
+RUN yarn --version
+RUN python3 --version
+RUN sudo aws --version
```

Hadolint

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)